custom/conf/app.example.ini

```diff
@@ -827,6 +827,17 @@ LEVEL = Info
 ;; Dependencies can be added from any repository where the user is granted access or only from the current repository depending on this setting.
 ;ALLOW_CROSS_REPOSITORY_DEPENDENCIES = true
 ;;
+;; Enable button that redirects users to a map service in Location fields on profiles
+;ENABLE_LOCATION_BUTTON = true
+;;
+;; Default map service. No external API support has been included. A service has to allow
+;; searching using URL parameters. Only relevant if ENABLE_LOCATION_BUTTON is enabled.
+;; Some examples are:
+;; - https://www.google.com/maps/place/
+;; - https://www.mapquest.com/search/
+;; - https://www.bing.com/maps?where1=
+; LOCATION_QUERY_URL = https://www.openstreetmap.org/search?query=
+;;
```

Comment:
1. IMO these two variables just duplicates.
2. The config key could be `USER_LOCATION_MAP_URL`: this URL is not used for "query", but it's used for showing user's location on the map

If `USER_LOCATION_MAP_URL` is empty, do not show the link, otherwise, show the link

Then there won't be comment like "Checking whether .LocationQueryURL is empty is arguably excessive"